angle_min: 0.0
angle_max: 6.28318977356
angle_increment: 0.0175019223243
